src/dmd/globals.d

```diff
@@ -378,6 +378,8 @@ alias d_uns64 = uint64_t;
 // file location
 struct Loc
 {
+    // Note: it is assumed that filename is always either absolute or
+    //       relative to CWD
     const(char)* filename;
```

Comment:
Maybe you should make this a comment against the field, such as:
```
const(char)* filename;  // either absolute or relative to cwd
```
Make it a definitive comment, rather than a presumptuous one.

Reply:
ok